### ISSUE TITLE
English documentation is not in english

### ISSUE THREAD

**Issue by frob** [kakserpom/phpdaemon#236]
After the installation portion of the docs page the language switches from english to something else. 

For example:
"PHPDaemon представляет из себя один мастер-процесс с несколькими рабочими процессами" is not english even though the url (https://daemon.io/docs/en) is to an english page.

**Comment by kakserpom** [comment]
Right, we could use some help with that :)

**Comment by Guyverix** [comment]
It is NOT a good solution, but it helped me a bit to learn how the system is glued together:

https://translate.google.com/translate?hl=en&sl=ru&tl=en&u=https%3A%2F%2Fdaemon.io%2Fdocs%2Fen%2F%23control&sandbox=1

I forced google translate to change it over after it insisted it was in English already, I told it in the dropdown Russian, and it translated the page (I hope) properly.

**Comment by bobboau** [comment]
It is rather hard to figure out how to use this library when all the documentation from the "Basics" section onwards is in Russian. despite the fact that this library seems to have everything I need listed in it's feature set.

**Comment by rgpublic** [comment]
Yes, I just found out about phpdaemon.io and it certainly looks impressive. Exactly what I was searching for. Supports ChRoot, ChDir, Configurable Application Routing... Brilliant! But without a major part of the docs in a readable language this is sadly almost impossible to use :-( So, I started using Google Translate and DeepL and just created my first Pull Request :-) If this gets accepts by the maintainers, I hope I will be able to translate this step by step. There might be some problems along the way, because I don't personally understand the original version at all but currently with the "cryptic" letters it's completely undecipherable so any kind of translation would certainly be a huge step forward.

**Comment by kakserpom** [comment]
@rgpublic are you on telegram? `@systemofcells`
